public/schema.graphql

```diff
@@ -270,6 +284,27 @@ enum NewsItemsOrder {
   updatedAt_DESC
 }
 
+type OpenWeatherMap {
+  alerts: [String!]
+  createdAt: String
+  current: String
+  daily: [String!]
+  hourly: [String!]
```

Comment:
Hier liegen (in der response später) JSON-Einträge, welche du so von der OWM-API bekommst, und cachest?

Reply:
ich denke die Schemas sind nur GraphQl Clients um auslesen zu können was die API kann